style.css

```diff
@@ -0,0 +1,55 @@
+* {
+  background-color: gainsboro;
+}
+div {
+  text-align: center;
+}
+
+body {
+  font-family: "Nunito", sans-serif;
+}
+header {
+  font-size: 26px;
+  line-height: 10px;
+  color: rgb(255, 255, 255);
+  text-align: center;
```

Comment:
This is duplicating the `text-align` rule you have on line 5 and is unnecessary.
```suggestion
```